src/main/java/io/dropwizard/kafka/KafkaTopicsFactory.java

```diff
@@ -0,0 +1,61 @@
+package io.dropwizard.kafka;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.jackson.Discoverable;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.Min;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class KafkaTopicsFactory extends ArrayList<KafkaTopicsFactory.KafkaTopic> implements Discoverable {
```

Comment:
This probably also doesn't need to be `Discoverable`. That's usually only useful when you have multiple implementations of a factory that you want to be able to switch to based on some value in the YAML, and not (ideally) having to change the code.